integration.js

```diff
@@ -119,6 +159,51 @@ async function downloadThumbnail(authToken, thumbnailUrl) {
   return `data:image/png;charset=utf-8;base64,${Buffer.from(response.data, 'binary').toString('base64')}`;
 }
 
+const MIME_EXPORT_TYPES = {
+  'application/vnd.google-apps.presentation': 'application/vnd.oasis.opendocument.presentation',
+  'application/vnd.google-apps.spreadsheet': 'text/csv',
+  'application/vnd.google-apps.document': 'application/vnd.oasis.opendocument.text',
+  'application/pdf': 'application/vnd.oasis.opendocument.presentation',
+  'application/vnd.google-apps.script': 'application/vnd.google-apps.script+json'
+};
```

Comment:
Are there very specific file formats that we support displaying content for?  If we have a known list where it works I think we should add that to the README

Reply:
We cover all types listed as possible text export types listed on google drives docs https://developers.google.com/drive/api/guides/ref-export-formats.